### ISSUE TITLE
[BUG]: Dashboard Onboarding "Run your first pipeline" instructions have nonexistent argument.

### ISSUE THREAD

**Issue by nickschuetz** [zenml-io/zenml#2989]
### System Information

# zenml info -a -s
ZENML_LOCAL_VERSION: 0.65.0
ZENML_SERVER_VERSION: 0.65.0
ZENML_SERVER_DATABASE: sqlite
ZENML_SERVER_DEPLOYMENT_TYPE: other
...
PYTHON_VERSION: 3.9.18
ENVIRONMENT: native
SYSTEM_INFO: {'os': 'linux', 'linux_distro': 'rhel', 'linux_distro_like': 'fedora', 'linux_distro_version': '9.4'}
ACTIVE_WORKSPACE: default
ACTIVE_STACK: default
ACTIVE_USER: openshift
TELEMETRY_STATUS: enabled
ANALYTICS_CLIENT_ID: 8f18cc9a-f836-48a8-be1f-371acde396b8
ANALYTICS_USER_ID: 3e58b065-0b56-4a3c-a1c8-a2e1ecff8487
ANALYTICS_SERVER_ID: d320735c-5519-41eb-bfb6-c28ee55549f1
INTEGRATIONS: ['airflow', 'bitbucket', 'kaniko', 'pillow', 'pytorch', 'pytorch_lightning', 'scipy', 'sklearn']
PACKAGES: {'markupsafe': '2.1.4', 'pymupdf': '1.23.25', 'pymupdfb': '1.23.22', 'pyyaml': '6.0.1', 'sqlalchemy': '2.0.27', 'aiohttp': '3.9.3', 'argon2-cffi-bindings': '21.2.0', 'bcrypt': '4.0.1', 'blis': '0.7.11', 'charset-normalizer': '3.2.0', 'chroma-hnswlib': '0.7.3', 'contourpy':
'1.2.0', 'cymem': '2.0.8', 'debugpy': '1.6.7.post1', 'diambra': '0.0.17', 'fonttools': '4.50.0', 'frozenlist': '1.4.1', 'greenlet': '2.0.2', 'grpcio': '1.62.1', 'h5py': '3.10.0', 'httptools': '0.6.0', 'jiter': '0.5.0', 'kiwisolver': '1.4.5', 'matplotlib': '3.8.3', 'ml-dtypes':
'0.2.0', 'mmh3': '4.1.0', 'multidict': '6.0.5', 'murmurhash': '1.0.10', 'numpy': '1.26.4', 'onnxruntime': '1.17.1', 'opencv-python': '4.9.0.80', 'opencv-python-headless': '4.9.0.80', 'orjson': '3.10.7', 'pandas': '2.2.1', 'pillow': '10.2.0', 'preshed': '3.0.9', 'protobuf': '3.20.3',
'psutil': '5.9.5', 'pulsar-client': '3.4.0', 'pyarrow': '17.0.0', 'pydantic-core': '2.18.4', 'pyzmq': '25.1.1', 'regex': '2023.12.25', 'rpds-py': '0.10.0', 'safetensors': '0.4.4', 'scikit-image': '0.22.0', 'scikit-learn': '1.4.0', 'scipy': '1.12.0', 'sentencepiece': '0.2.0',
'spacy': '3.7.2', 'srsly': '2.4.8', 'tensorflow': '2.15.0.post1', 'tensorflow-io-gcs-filesystem': '0.35.0', 'thinc': '8.2.2', 'tiktoken': '0.6.0', 'tokenizers': '0.15.2', 'torch': '2.2.0', 'torchvision': '0.17.0', 'tornado': '6.3.3', 'triton': '2.2.0', 'uvloop': '0.17.0',
'watchfiles': '0.21.0', 'websockets': '11.0.3', 'wrapt': '1.14.1', 'xxhash': '3.5.0', 'yarl': '1.9.4', 'authlib': '1.2.1', 'babel': '2.12.1', 'deprecated': '1.2.14', 'farama-notifications': '0.0.4', 'gitpython': '3.1.32', 'mako': '1.2.4', 'markdown': '3.5.2', 'pyjwt': '2.7.0',
'pymysql': '1.1.1', 'pypika': '0.48.9', 'pygments': '2.16.1', 'sqlalchemy-utils': '0.38.3', 'send2trash': '1.8.2', 'absl-py': '2.1.0', 'accelerate': '0.34.0', 'aiosignal': '1.3.1', 'albumentations': '1.3.1', 'alembic': '1.8.1', 'allegroai': '3.10.1', 'analytics-python': '1.4.post1',
'annotated-types': '0.6.0', 'anthropic': '0.28.1', 'anyio': '4.3.0', 'argon2-cffi': '23.1.0', 'arrow': '1.2.3', 'asgiref': '3.7.2', 'asttokens': '2.2.1', 'astunparse': '1.6.3', 'async-lru': '2.0.4', 'async-timeout': '4.0.3', 'attrs': '23.1.0', 'azure-common': '1.1.28', 'azure-core':
'1.29.3', 'azure-mgmt-core': '1.4.0', 'azure-mgmt-resource': '23.0.1', 'backcall': '0.2.0', 'backoff': '1.10.0', 'beautifulsoup4': '4.12.3', 'bleach': '6.0.0', 'boto3': '1.34.153', 'botocore': '1.34.153', 'bs4': '0.0.2', 'build': '1.0.3', 'cachetools': '5.3.2', 'catalogue':
'2.0.10', 'certifi': '2024.2.2', 'chromadb': '0.4.24', 'click-params': '0.3.0', 'cloudpathlib': '0.16.0', 'cloudpickle': '2.2.1', 'colorama': '0.4.6', 'coloredlogs': '15.0.1', 'comm': '0.1.4', 'commonmark': '0.9.1', 'confection': '0.1.4', 'cycler': '0.12.1', 'dacite': '1.8.1',
'dataclasses-json': '0.6.4', 'datasets': '2.21.0', 'defusedxml': '0.7.1', 'diambra-arena': '2.2.6', 'diambra-engine': '2.2.1', 'dill': '0.3.8', 'dirtyjson': '1.0.8', 'distro': '1.9.0', 'docker': '7.1.0', 'docopt': '0.6.2', 'donkeycar': '5.0.0', 'ecdsa': '0.18.0', 'exceptiongroup':
'1.1.3', 'executing': '1.2.0', 'fastai': '2.7.14', 'fastapi': '0.110.0', 'fastapi-utils': '0.2.1', 'fastcore': '1.5.29', 'fastdownload': '0.0.7', 'fastjsonschema': '2.18.0', 'fastprogress': '1.0.3', 'filelock': '3.13.1', 'flatbuffers': '23.5.26', 'fqdn': '1.5.1', 'fsspec':
'2023.12.2', 'furl': '2.1.3', 'gast': '0.5.4', 'gitdb': '4.0.10', 'google-auth': '2.27.0', 'google-auth-oauthlib': '1.2.0', 'google-pasta': '0.2.0', 'googleapis-common-protos': '1.62.0', 'gymnasium': '0.29.1', 'h11': '0.14.0', 'httpcore': '1.0.4', 'httplib2': '0.19.1', 'httpx':
'0.27.0', 'huggingface-hub': '0.21.2', 'humanfriendly': '9.2', 'idna': '3.6', 'imageio': '2.33.1', 'importlib-metadata': '6.8.0', 'importlib-resources': '6.1.1', 'inputs': '0.5', 'ipinfo': '5.1.1', 'ipykernel': '6.25.1', 'ipython': '8.14.0', 'ipython-genutils': '0.2.0',
'ipywidgets': '7.8.0', 'isodate': '0.6.1', 'isoduration': '20.11.0', 'jedi': '0.19.0', 'jmespath': '1.0.1', 'joblib': '1.3.2', 'json5': '0.9.14', 'jsonpointer': '2.4', 'jsonschema': '4.19.0', 'jsonschema-specifications': '2023.7.1', 'jupyter-client': '8.3.1', 'jupyter-core':
'5.3.1', 'jupyter-events': '0.7.0', 'jupyter-lsp': '2.2.0', 'jupyter-server': '2.7.2', 'jupyter-server-terminals': '0.4.4', 'jupyterlab': '4.0.5', 'jupyterlab-pygments': '0.2.2', 'jupyterlab-server': '2.24.0', 'jupyterlab-widgets': '1.1.5', 'keras': '2.15.0', 'kubernetes': '29.0.0',
'langcodes': '3.3.0', 'lazy-loader': '0.3', 'libclang': '16.0.6', 'lightning-utilities': '0.10.1', 'llama-index': '0.10.14', 'llama-index-agent-openai': '0.1.5', 'llama-index-cli': '0.1.6', 'llama-index-core': '0.10.60', 'llama-index-embeddings-openai': '0.1.6',
'llama-index-indices-managed-llama-cloud': '0.1.3', 'llama-index-legacy': '0.9.48', 'llama-index-llms-anthropic': '0.1.16', 'llama-index-llms-bedrock': '0.1.12', 'llama-index-llms-groq': '0.1.4', 'llama-index-llms-ollama': '0.2.2', 'llama-index-llms-openai': '0.1.6',
'llama-index-llms-openai-like': '0.1.3', 'llama-index-multi-modal-llms-openai': '0.1.4', 'llama-index-program-openai': '0.1.4', 'llama-index-question-gen-openai': '0.1.3', 'llama-index-readers-file': '0.1.6', 'llama-index-readers-llama-parse': '0.1.3',
'llama-index-vector-stores-chroma': '0.1.5', 'llama-index-vector-stores-redis': '0.1.2', 'llama-parse': '0.3.4', 'llamaindex-py-client': '0.1.13', 'loguru': '0.7.2', 'luqum': '0.11.0', 'markdown-it-py': '3.0.0', 'marshmallow': '3.21.0', 'matplotlib-inline': '0.1.6', 'mdurl':
'0.1.2', 'mistune': '3.0.1', 'monotonic': '1.6', 'mpmath': '1.3.0', 'multiprocess': '0.70.16', 'mypy-extensions': '1.0.0', 'nbclient': '0.8.0', 'nbconvert': '7.8.0', 'nbformat': '5.9.2', 'nest-asyncio': '1.6.0', 'networkx': '3.2.1', 'nltk': '3.8.1', 'notebook': '7.0.3',
'notebook-shim': '0.2.3', 'nvidia-cublas-cu12': '12.1.3.1', 'nvidia-cuda-cupti-cu12': '12.1.105', 'nvidia-cuda-nvrtc-cu12': '12.1.105', 'nvidia-cuda-runtime-cu12': '12.1.105', 'nvidia-cudnn-cu12': '8.9.2.26', 'nvidia-cufft-cu12': '11.0.2.54', 'nvidia-curand-cu12': '10.3.2.106',
'nvidia-cusolver-cu12': '11.4.5.107', 'nvidia-cusparse-cu12': '12.1.0.106', 'nvidia-nccl-cu12': '2.19.3', 'nvidia-nvjitlink-cu12': '12.3.101', 'nvidia-nvtx-cu12': '12.1.105', 'oauthlib': '3.2.2', 'ollama': '0.3.1', 'openai': '1.14.2', 'opentelemetry-api': '1.23.0',
'opentelemetry-exporter-otlp-proto-common': '1.23.0', 'opentelemetry-exporter-otlp-proto-grpc': '1.23.0', 'opentelemetry-instrumentation': '0.44b0', 'opentelemetry-instrumentation-asgi': '0.44b0', 'opentelemetry-instrumentation-fastapi': '0.44b0', 'opentelemetry-proto': '1.23.0',
'opentelemetry-sdk': '1.23.0', 'opentelemetry-semantic-conventions': '0.44b0', 'opentelemetry-util-http': '0.44b0', 'opt-einsum': '3.3.0', 'orderedmultidict': '1.0.1', 'overrides': '7.4.0', 'packaging': '24.1', 'paho-mqtt': '1.6.1', 'pandocfilters': '1.5.0', 'parso': '0.8.3',
'passlib': '1.7.4', 'pathlib2': '2.3.7.post1', 'pexpect': '4.8.0', 'phospho': '0.3.33', 'pickleshare': '0.7.5', 'pip': '24.2', 'platformdirs': '3.10.0', 'ply': '3.11', 'posthog': '3.4.2', 'prettytable': '3.9.0', 'progress': '1.6', 'prometheus-client': '0.17.1', 'prompt-toolkit':
'3.0.39', 'ptyprocess': '0.7.0', 'pure-eval': '0.2.2', 'pyasn1': '0.5.0', 'pyasn1-modules': '0.3.0', 'pydantic': '2.7.4', 'pydantic-settings': '2.2.1', 'pyfiglet': '1.0.2', 'pynmea2': '1.19.0', 'pyparsing': '2.4.7', 'pypdf': '4.0.2', 'pyproject-hooks': '1.0.0', 'pyserial': '3.5',
'python-dateutil': '2.9.0.post0', 'python-dotenv': '1.0.1', 'python-jose': '3.3.0', 'python-json-logger': '2.0.7', 'python-multipart': '0.0.9', 'python-terraform': '0.10.1', 'pytorch-lightning': '2.1.4', 'pytz': '2024.1', 'qudida': '0.0.4', 'redis': '5.0.2', 'referencing': '0.30.2',
'requests': '2.32.3', 'requests-oauthlib': '1.3.1', 'rfc3339-validator': '0.1.4', 'rfc3986-validator': '0.1.1', 'rich': '13.7.1', 'rsa': '4.9', 's3transfer': '0.10.2', 'screeninfo': '0.8.1', 'secure': '0.3.0', 'setuptools': '69.2.0', 'shellingham': '1.5.4', 'simple-pid': '2.0.0',
'six': '1.16.0', 'smart-open': '6.4.0', 'smmap': '5.0.0', 'sniffio': '1.3.1', 'soupsieve': '2.4.1', 'spacy-legacy': '3.0.12', 'spacy-loggers': '1.0.5', 'sqlalchemy2-stubs': '0.0.2a35', 'sqlmodel': '0.0.18', 'stack-data': '0.6.2', 'starlette': '0.36.3', 'sympy': '1.12', 'tenacity':
'8.2.3', 'tensorboard': '2.15.1', 'tensorboard-data-server': '0.7.2', 'tensorflow-estimator': '2.15.0', 'termcolor': '2.4.0', 'terminado': '0.17.1', 'threadpoolctl': '3.2.0', 'tifffile': '2024.1.30', 'tinycss2': '1.2.1', 'tk': '0.1.0', 'tomli': '2.0.1', 'torchmetrics':
'1.3.0.post0', 'tqdm': '4.66.5', 'traitlets': '5.9.0', 'transformers': '4.39.3', 'typer': '0.9.0', 'typing-extensions': '4.10.0', 'typing-inspect': '0.9.0', 'tzdata': '2024.1', 'uri-template': '1.3.0', 'utm': '0.7.0', 'uvicorn': '0.27.1', 'validators': '0.18.2', 'wasabi': '1.1.2',
'watchgod': '0.8.2', 'wcwidth': '0.2.6', 'weasel': '0.3.4', 'weaviate-client': '3.24.1', 'webcolors': '1.13', 'webencodings': '0.5.1', 'websocket-client': '1.6.2', 'werkzeug': '3.0.1', 'wheel': '0.42.0', 'widgetsnbextension': '3.6.5', 'zenml': '0.65.0', 'zipp': '3.16.2',
'pygobject': '3.40.1', 'cffi': '1.15.0', 'cryptography': '36.0.1', 'dbus-python': '1.2.18', 'ethtool': '0.15', 'gpg': '1.15.1', 'libcomps': '0.1.18', 'libvirt-python': '10.0.0', 'lxml': '4.6.5', 'rpm': '4.16.1.3', 'selinux': '3.6', 'setools': '4.4.4', 'subscription-manager':
'1.29.40', 'systemd-python': '234', 'configargparse': '1.7', 'jinja2': '3.1.2', 'pysocks': '1.7.1', 'acme': '2.11.0', 'ansible-core': '2.14.14', 'certbot': '2.11.0', 'chardet': '4.0.0', 'click': '8.0.3', 'configobj': '5.0.6', 'dasbus': '1.4', 'decorator': '4.4.2', 'iniconfig':
'1.1.1', 'iniparse': '0.4', 'josepy': '1.13.0', 'nftables': '0.1', 'parsedatetime': '2.6', 'pluggy': '0.13.1', 'podman-compose': '1.0.6', 'py': '1.10.0', 'pyopenssl': '21.0.0', 'pyrfc3339': '1.1', 'pycparser': '2.21', 'pyinotify': '0.9.6', 'pytest': '6.2.2', 'resolvelib': '0.5.4',
'sepolicy': '3.6', 'setroubleshoot': '3.3.32', 'toml': '0.10.2', 'urllib3': '1.26.5'}

CURRENT STACK

Name: default
ID: b5e495cf-397c-4684-92cb-0060ad5d75d1
Workspace: default / 14867541-f839-4853-9c72-7b3ffc4d495d

ORCHESTRATOR: default

Name: default
ID: 2c29cc89-5e92-4fe9-9541-94a39118498c
Type: orchestrator
Flavor: local
Configuration: {}
Workspace: default / 14867541-f839-4853-9c72-7b3ffc4d495d

ARTIFACT_STORE: default

Name: default
ID: bd5bacb9-2d73-4816-8ede-d0cd1aa379d9
Type: artifact_store
Flavor: local
Configuration: {'path': ''}
Workspace: default / 14867541-f839-4853-9c72-7b3ffc4d495d

### What happened?

The dashboard indicates to run the following to kick off the quickstart pipeline:

python run.py --training-pipeline

However, `--training-pipeline` is not a valid argument.

```
# python run.py --training-pipeline
The cache for model files in Transformers v4.22.0 has been updated. Migrating your old cache. This is a one-time only operation. You can interrupt this and resume the migration later on by calling `transformers.utils.move_cache()`.
0it [00:00, ?it/s]
PyTorch version 2.2.0 available.
TensorFlow version 2.15.0.post1 available.
Usage: run.py [OPTIONS]
Try 'run.py --help' for help.

Error: No such option: --training-pipeline
```

### Reproduction steps

1. Install ZenML
2. Run the quickstart pipeline per the Dashboard Onboarding instructions.
3.
...

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nickschuetz** [comment]
https://github.com/zenml-io/zenml/blob/main/examples/quickstart/run.py
